etc/kapacitor/kapacitor.conf

```
@@ -370,6 +370,8 @@ default-retention-policy = ""
   type = "rpc"
   # Temporary request transaction ID.
   tid = 1
+  # Collector name.
+  # collector = ""
```

A reasonable default could be provided
```suggestion
collector = "Kapacitor"
```